src/components/layout/nav/PrivateMenu.tsx

```diff
@@ -52,34 +53,41 @@ export default function PrivateMenu() {
   const { data: session, status } = useSession()
   const router = useRouter()
   const [anchorEl, setAnchorEl] = useState<Element | null>(null)
+  const { data: currentPerson } = useCurrentPerson()
+  const userPicture = currentPerson?.user.picture
 
   const handleMenu = (event: React.MouseEvent) => setAnchorEl(event.currentTarget)
   const handleClose = () => setAnchorEl(null)
+  const getAvatar = () => {
+    if (userPicture) return userPicture as string
+  }
```

Comment:
This function seems to be redundant, no need to wrap it as you can use directly `userPicture` variable.